locale/en/manageiq.po

```diff
@@ -39402,6 +39404,10 @@ msgstr ""
 msgid "Invalid button action"
 msgstr ""
 
+#: ../app/controllers/service_controller.rb
+msgid "Invalid button action!"
```

Comment:
Minor, but should this match the one above ☝️

Reply:
Oh yeah, one has a `!` - probably should be changed to match. good 👀

Reply:
@jeffibm @DavidResende0 can you take a look at normalizing these values?  It's very low urgency.

Reply:
@jrafanie https://github.com/ManageIQ/manageiq-ui-classic/pull/9012

Reply:
Merged